airflow_requirements.txt

```diff
@@ -8,4 +8,5 @@ pydantic==1.10.9
 # Data sources requirements
 apache-airflow-providers-apache-drill==2.4.3
 google-cloud-bigquery==3.11.3
+google-ads
```

Comment:
Are you explicitly choosing not to add a specific version for this? Consider adding at least a compatible release requirement, e.g. google-ads==22.* or google-ads>=22.0

Reply:
Done, but the reason I didn't set it as I was getting some conflicts trying to generate the fixed package numbers. Let's test tomorrow to see if this works.

Reply:
Should be fixed now, but still seeing another integration test failure. Looks like it can't find integration_test.csvh.